x/nft/CONTRACT.md

```diff
@@ -0,0 +1,4 @@
+# Contract
+
```

Comment:
@fedekunze @okwme @rigelrozanski Would like input here on what we would typically expect and/or want in a soft contract for module ownership and maintenance.

Reply:
* Review issues and pull requests related to the module
 * Resolve issues and pull requests within 90 days
 * Ensure module is ready for SDK releases

Reply:
Updated :)